internal/webserver/webserver.go

```diff
@@ -97,6 +101,8 @@ type kubeFilter struct {
 	skipImpersonationReview    bool
 	reverseProxy               *httputil.ReverseProxy
 	bearerToken                string
+	bearerTokenFile            string
+	bearerTokenExpirationTime  int64
```

Comment:
couldn't we easily store `time.Time`, rather than an `int64`?

Reply:
fixed